js/foundation.offcanvas.js

```diff
@@ -146,6 +147,16 @@ class OffCanvas {
   }
 
   /**
+   * Stops scrolling of the body when offcanvas is open on mobile Safari and other troublesome browsers.
+   * @private
+   */
+  _stopScrolling(event) {
+  	event.preventDefault();
+  	event.stopPropagation();
+  	return false;
```

Comment:
Will change - will need someone to check on mobile Safari to confirm though please (no longer have access to check)